vagrant/provisioning/roles/alfresco/tasks/main.yml

```diff
@@ -374,6 +374,60 @@
       opencmis.server.override=true
       opencmis.server.value=https://{{ external_host }}/alfresco/api
 
+- name: change alfresco-global.properties to support second LDAP configuration
+  block:
+    - name: add ldap2 in AD/Samba section
+      become: yes
+      become_user: alfresco
+      replace:
+        path: "{{ root_folder }}/app/alfresco/shared/classes/alfresco-global.properties"
+        regexp: authentication\.chain=external1:external,ldap1:ldap-ad,alfrescoNtlm1:alfrescoNtlm
+        replace: authentication.chain=external1:external,ldap1:ldap-ad,ldap2:ldap-ad,alfrescoNtlm1:alfrescoNtlm
+    - name: remove LDAP host-specific settings section
+      become: yes
+      become_user: alfresco
+      replace:
```

Comment:
Why not just remove this part of the earlier alfresco-global.properties template?  Why should the alfresco role first add LDAP host-specific settings, and then remove them?

Reply:
From the comments bellow, if we always create the ldap1 folder than i will made a changes according to that

Reply:
@kikomanaskov333, exactly; we always create ldap1 and put the ArkCase LDAP info there; so we never need the LDAP host-specific info in the alfresco-global.properties.  In case foia_portal_context is defined, then we also create an ldap2 folder, but we always create ldap1.